webpack.common.js

```diff
@@ -105,16 +105,6 @@ let config = {
         require.resolve(basePath + '/app/matomo' + MATOMO_VERSION + '.html')
       )
     }),
-    new HtmlWebpackPlugin({
```

Comment:
I feel like applying this:
https://github.com/survivejs/webpack-merge#uniquefield-fields-field--field
right away instead of going back and forth on it is probably more efficient

Reply:
fiiiiine